mobile/Components/MapCard.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  StyleSheet, View, Image, Text, Dimensions,
+} from 'react-native';
+
+const { height } = Dimensions.get('window');
+const CARD_HEIGHT = height / 4;
+const CARD_WIDTH = CARD_HEIGHT + 50;
+
+const styles = StyleSheet.create({
+  card: {
+    padding: 10,
+    borderRadius: 8,
+    elevation: 2,
+    backgroundColor: '#FFF',
+    marginHorizontal: 10,
+    shadowColor: '#000',
+    shadowRadius: 5,
+    shadowOpacity: 0.3,
+    shadowOffset: { x: 2, y: -2 },
+    height: CARD_HEIGHT,
+    width: CARD_WIDTH,
+    overflow: 'hidden',
+  },
+  cardImage: {
+    flex: 3,
+    borderRadius: 8,
+    width: '100%',
+    height: '100%',
+    alignSelf: 'center',
+  },
+  textContent: {
+    flex: 1,
+  },
+  cardTitle: {
+    fontSize: 12,
+    marginTop: 5,
+    fontWeight: 'bold',
+  },
+  cardDescription: {
+    fontSize: 12,
+    color: '#444',
+  },
+});
+
+function MapCard({
+  id, image, title, description
```

Comment:
Props need to be validated with `import PropTypes from 'prop-types';` to specify what type they are expecting in the component